### PR TITLE
Remove border for prefix and suffix and add mod-alignRight

### LINKED ISSUE #2781: [Textfield] suffix, prefix and text align
Few improvements are required on textfield: 
- Remove prefix & suffix border (without any interaction changes)
- Add a mod to align text on right

Frame 6 (1)

### PR DESCRIPTION
**Pull request by vvalentin-lucca**
## Description

Fix #2781 

-----

-----

Capture d’écran 2024-05-22 à 12 12 32

Capture d’écran 2024-05-22 à 12 12 11

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2810/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/17302/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2810/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/17324/ui-diff)

**Review by Supamiu — APPROVED** [review]

File: packages/ng/forms/text-input/text-input.component.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="textField">
+<div class="textField" [class.mod-alignRight]="alignRight">
```
Comment (fbasmaison-lucca) [inline packages/ng/forms/text-input/text-input.component.html:1]: I’m wondering if specifying what we’re aligning right would be a good thing. What if we want to align the help text to the right of the container later ?

Maybe `valueAlignRight`?

File: packages/scss/src/components/textField/component.scss [hunk 2]
```diff
@@ -18,7 +19,7 @@
 			display: flex;
 			align-items: center;
 			width: 100%;
-			box-shadow: 0 0 0 1px var(--component-textField-border);
+			// box-shadow: 0 0 0 1px var(--component-textField-border);
```
Comment (fbasmaison-lucca) [inline packages/scss/src/components/textField/component.scss:22]: Why not removing these lines?

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2810/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/17472/ui-diff)

**Review by GuillaumeNury — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2810/storybook

**Review by jeremie-lucca — APPROVED** [review]

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/17474/ui-diff)

**Pull request merged**